docs/releases/core/release-notes.md

```diff
@@ -6,8 +6,250 @@ hide_title: false
 
 While the Vega core software is closed-source, you can refer here for a full list of release notes for each version that the validators use to run the Vega mainnet. Releases are listed with their semantic version number and the date the release was made available to mainnet validators.
 
-### Versions 0.46.0-0.47.6 combined | 2022-01-11
+### Versions 0.50.3-0.49.8 combined | 2022-04-27
+This release was made available to validators on 27 April, 2022.
+
+The primary focus of this and the next upcoming releases has been to complete the final remaining features, progress data-node improvements for scalability and to add test coverage and fix bugs.
+
+Note: While many of the features below are related to trading, it is not yet enabled on mainnet. 
+
+**Proposals to change market parameters**: After a market has been proposed and enacted, changes to the market parameters are required. Tokenholders will be able to submit proposals to change market parameters.
```

Comment:
> "After a market has been proposed and enacted, changes to the market parameters are required."

They are? Isn't this more of an 'IF' changes are required, then...? Feel free to close if I'm wrong.

Reply:
Not sure how that slipped by - you're right, thanks!